src/components/TwitterChatter/TwitterChatter.vue

```diff
@@ -47,19 +47,19 @@ export default {
       screenname: '',
       options: [
         {
-          value: 'DonaldTrump',
+          value: 'realDonaldTrump',
           label: 'Donald Trump',
-          name: 'DonaldTrump'
+          name: 'realDonaldTrump'
         },
         {
           value: 'JoeBiden',
           label: 'Joe Biden',
           name: 'JoeBiden'
         },
         {
-          value: 'Mike_Pense',
-          label: 'Mike Pense',
-          name: 'Mik_Pense'
+          value: 'VP',
```

Comment:
should be Mike_Pence instead of VP, as the VP account is rotated to the current VP correct?

Reply:
consider as a separate issue